packages/common/src/types/filterGrammar.test.ts

```diff
@@ -90,6 +90,20 @@ describe('attachTypesToModels', () => {
         expect(parser.parse(' >=32')).toEqual(expected);
         expect(parser.parse(' >= 32')).toEqual(expected);
     });
+
+    it('Is null', async () => {
+        expect(parser.parse('NULL')).toEqual({
+            is: true,
+            type: 'null',
+        });
+    });
+
+    it('Is not null', async () => {
+        expect(parser.parse('!null')).toEqual({
```

Comment:
Should it work for both lower and uppercase?

Reply:
yes, because of this lowercase https://github.com/lightdash/lightdash/pull/5729/files#diff-546a539fae0c7fc3cf99abec54c5cb175292ec43d5f4525a15bdb84080ba0be5R181